fim/graph/abc_property_graph_constants.py

```diff
@@ -58,10 +59,13 @@ class ABCPropertyGraphConstants(ABC):
     PROP_DETAILS = 'Details'
     PROP_RESERVATION_INFO = 'ReservationInfo'
     PROP_NODE_MAP = "NodeMap"
+    PROP_STRUCTURAL_INFO = "StructuralInfo"
+    PROP_STITCH_NODE = "StitchNode"
     # these properties get validated to be valid JSON objects whenever someone validates the graph
     JSON_PROPERTY_NAMES = [PROP_LABELS, PROP_CAPACITIES, PROP_LABEL_DELEGATIONS,
                            PROP_CAPACITY_DELEGATIONS, PROP_LABEL_ALLOCATIONS,
-                           PROP_CAPACITY_ALLOCATIONS, PROP_RESERVATION_INFO]
+                           PROP_CAPACITY_ALLOCATIONS, PROP_RESERVATION_INFO,
+                           PROP_RESERVATION_INFO, PROP_STRUCTURAL_INFO]
```

Comment:
PROP_RESERVATION_INFO is duplicated here.

Reply:
Added a fix